share/oaipmh/util.py

```diff
@@ -0,0 +1,3 @@
+
+def format_datetime(dt):
+    return dt.strftime('%Y-%m-%dT%H:%M:%SZ')
```

Comment:
OAI is [picky](https://www.openarchives.org/OAI/openarchivesprotocol.html#Dates) about their dates:
> the "Complete date plus hours, minutes, and seconds" variant of ISO8601. This format is YYYY-MM-DDThh:mm:ssZ.

> When time is included, the special UTC designator ("Z") must be used.

Explicitly formatting it seemed less hacky than isoformat, chop off the end, and add a Z.